### PR TITLE
Sprint 3

### PR DESCRIPTION
**Pull request by GregoryForrest**
Add Technical Order Counts
Add In Acquisition Counts
Add Configuration Plan
Update Labor Cost
Add Distribution Cost
Add Contractor Support Questions
Update how useWatch is typed for better type support
Add TOAP Migration
Add Additional Labor
Add System/Mission Description
Move Labor Type and associated info to Labor page
Move GAL button to left for TO Manager like Program Manager
Move the "Add Program Manager" button to the left similar to GAL

Generated Word Document Updates:
Section 1.2 Sustainment Costs
Section 1.3 TOAP Migration Plan
Section 1.5 Acquisition of New TOs
Section 1.6 Configuration Plan
Section 1.7 Program TOs
Section 1.8 System/Mission Description
Reduced gap between PM and TO Manager on cover page
Centered page numbers

### CODE REVIEW

File: src/Steps/TechnicalOrders/Fields/TOCounts.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <fieldset>
+      <legend>
+        <Text weight="semibold">Number of Technical Orders</Text>
+      </legend>
+      <div className="requestFieldContainer">
+        <BACInput<CAFTOPTechnicalOrders>
+          name="NumElectronic"
+          labelText="Electronic WA-1 or WA-2"
+          rules={{ required: true }}
+          fieldProps={{ type: "number" }}
```
Comment (RobertPorterfield) [inline src/Steps/TechnicalOrders/Fields/TOCounts.tsx]: I believe you can also set the min here so the decrement button hard stops at 0.
`fieldProps={{ type: "number", min: 0 }}`

Comment (RobertPorterfield) [reply src/Steps/TechnicalOrders/Fields/TOCounts.tsx]: Obviously same goes for the other 4 `type: "number"` fields

File: src/Steps/Distribution/Fields/DistCost.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            name="DistCost"
+            labelText="Distribution Cost"
+            rules={{ required: true }}
+            fieldProps={{
+              type: "number",
+              step: "1",
+              min: "0",
+              contentBefore: <Text>$</Text>,
+            }}
+          />
```
Comment (RobertPorterfield) [inline src/Steps/Distribution/Fields/DistCost.tsx:88]: Matter of opinion here, so take with a grain of salt...
I'd leave the Distribution Cost visible, but disable/enable based on the yes/no answer. This prevents the form buttons from jumping up/down the screen. Consider resetting the DistCost field to empty input when selection is changed to no.

File: src/Steps/TechnicalOrders/Fields/TOCounts.tsx (hunk 1 above)
Comment (GregoryForrest) [reply src/Steps/TechnicalOrders/Fields/TOCounts.tsx]: Great suggestion! Implemented on all 5 fields.

File: src/Steps/Complete.tsx [hunk 3]
```diff
@@ -35,6 +45,63 @@ const Complete = (props: ICAFTOPWizardStep) => {
       ".docx",
     ].join("_");
 
+    const totalCount =
+      (globalState.TechnicalOrders.NumAuthoredInTOAP !== ""
+        ? globalState.TechnicalOrders.NumAuthoredInTOAP
+        : 0) +
```
Comment (RobertPorterfield) [inline src/Steps/Complete.tsx:51]: Not that this is wrong, but would it be clearer to make all these `myVar || 0` ?

**Review by RobertPorterfield — APPROVED** [review]
Excellent work as always!

**Pull request merged**